packages/components/src/internal/util/global.ts

```diff
@@ -13,34 +13,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { getGlobal, setGlobal } from 'reactn';
+// import { getGlobal, setGlobal } from 'reactn';
 import createHistory from 'history/createBrowserHistory';
 
+let _BrowserHistory;
+
 /**
  * Initialize the global state object for this package.
  */
 export function initBrowserHistoryState() {
-    if (!getGlobal()['BrowserHistory']) {
-        setGlobal(
-            {
-                BrowserHistory: createHistory(),
-            },
+    _BrowserHistory = createHistory();
 
-            global => {
-                // add a no-op listener just to connect this global state history to the url changes
-                getBrowserHistory().listen((location, action) => {});
-            }
-        );
-    }
+    // add a no-op listener just to connect this global state history to the url changes
+    _BrowserHistory.listen((location, action) => {});
```

Comment:
I believe we can remove this, this was needed for ReactN.

Reply:
done

Reply:
Turns out we do need this! After I removed it, some clicks/links in the app no logger triggered route changes. Added back in and things started working again as expected.